dependent/plastic-and-reconstructive-surgery.csl

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US">
+  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/npg -->
```

Comment:
@damnation333, we use these XML comments in automatically generated dependent styles to keep track from which metadata folder at https://github.com/citation-style-language/journals they originated. It should be absent from all manually edited styles.